### PR TITLE
data.json

### PR DESCRIPTION
**Pull request by CodeSizzler**

### CODE REVIEW

File: 2023/DearAzure/data.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "kasamshaikh",
+      "",
```
Comment (noopman) [inline 2023/DearAzure/data.json:6]: You need to add your handle @CodeSizzler here, to make you a valid maintainer of this folder!

File: 2023/DearAzure/data.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      ""
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Dear Azure",
+    "Country": "INDIA",
+    "Url": "https://www.eventbrite.com/o/dear-azure-az-india-by-kasam-shaikh-18178264307",
+    "Description_SessionizeApiId": "If you have configured your Sessionize correctly then browsing to this url (replacing {Your sessionize event Id} with your event sessionize id) will return valid json: https://sessionize.com/api/v2/{Your sessionize event Id}/view/Sessions",
+    "SessionizeApiId": "{Your sessionize event Id}",
```
Comment (noopman) [inline 2023/DearAzure/data.json:15]: Will you be using Sessionize to show off your event sessions on globalazure.net?

File: 2023/DearAzure/data.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    "SessionizeEventName": "https://www.eventbrite.com/o/dear-azure-az-india-by-kasam-shaikh-18178264307",
+    "Logo": {
+      "Description": "Community logo is a .png file 500X500px in this folder!",
+      "File": "dearazure.png"
```
Comment (noopman) [inline 2023/DearAzure/data.json:19]: You need to add your logo file to the same folder and name it dearazure.png!

**Review by noopman — CHANGES_REQUESTED** [review]
I am missing an index.md file in this same folder! Look at other folders next to this one and get inspiration for your own index.md!

**Comment by tiagocostapt** [comment]
@CodeSizzler any updates? Do you still want to update your event with this PR?

**Pull request closed without merging**